README.md

```diff
@@ -22,11 +22,11 @@ Lighthouse is:
 
 - Ready for use on Ethereum consensus mainnet.
 - Fully open-source, licensed under Apache 2.0.
-- Security-focused. Fuzzing techniques have been continuously applied and several external security reviews have been performed.
+- Security-focused. Fuzzing techniques have been continuously applied, and several external security reviews have been performed.
 - Built in [Rust](https://www.rust-lang.org), a modern language providing unique safety guarantees and
 	excellent performance (comparable to C++).
-- Funded by various organisations, including Sigma Prime, the
-	Ethereum Foundation, ConsenSys, the Decentralization Foundation and private individuals.
+- Funded by various organizations, including Sigma Prime, the
```

Comment:
Most of the Lighthouse team are based in the `en_AU`-locale, and we're committed to using a mix of UK/US English. Where a concept exists in the Ethereum consensus spec we will always use US English, but otherwise it's a mix.

Reply:
Oh, I'm sorry, you have a good point.